macros: Fix load order of `ocil_fix_srg_privileged_command`

yuumasato opened the pull request:
#### Description:

- Relocate the `ocil_fix_srg_privileged_command`  macro so that it is loaded later, it needs other macros that need to be loaded before it.
- This is broken since introduction of `ocil_fix_srg_privileged_command` in https://github.com/ComplianceAsCode/content/pull/8807

#### Rationale:

- Macros that use other macros need to be loaded after them.

#### Review Hints:

- Try to automatus a rule that uses the macro:
```
python3  tests/automatus.py rule --libvirt qemu:///session rhel8 --datastream build/ssg-rhel8-ds.xml audit_rules_privileged_commands_kmod
Setting console output to log level INFO
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/wsato/git/content/logs/rule-custom-2022-11-21-1511/test_suite.log
ERROR - Terminating due to error: Error loading a Rule from /home/wsato/git/content/linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_kmod/rule.yml: 'complete_ocil_entry_audit_privileged_commands' is undefined.
WARNING - Nothing has been tested!
```

marcusburghardt commented:
/test e2e-aws-rhcos4-e8

marcusburghardt reviewed (changes requested):
This fixed the issue mentioned in #9824 . Thanks.

Initially it was not clear to me the intention of the new macro file: `20-composite.jinja`.
It is basically intended to contain macros related to multiple fields, like OCIL, fixtext and SRG. We already have specific macro files for each of these fields but not one for cases where multiple fields are reasonable in a macro.

This makes sense. However, the file intention is not totally clear in the first glance. Could you update the documentation about it, please?

yuumasato commented:
> This fixed the issue mentioned in #9824 . Thanks.
> 
> Initially it was not clear to me the intention of the new macro file: `20-composite.jinja`. It is basically intended to contain macros related to multiple fields, like OCIL, fixtext and SRG. We already have specific macro files for each of these fields but not one for cases where multiple fields are reasonable in a macro.
> 
> This makes sense. However, the file intention is not totally clear in the first glance. Could you update the documentation about it, please?

I added a comment in the file, not sure how that will be rendered in the readthedocs page.

I also renamed the file, and changed it to `20-rules`.

marcusburghardt commented:
/retest

yuumasato commented:
/retest

marcusburghardt reviewed (approved):
Thanks @yuumasato . I believe it would be good in the stabilization branch too.
